src/models/environment.ts

```diff
@@ -371,10 +371,10 @@ export class EnvironmentTreeItem extends TreeItem {
     // "ccloud-environment", "direct-environment", "local-environment"
     this.contextValue = contextParts.join("-");
 
-    if (isDirect(resource)) {
-      // mainly to help E2E tests distinguish direct connections from other tree items
-      this.accessibilityInformation = { label: `Direct connection: "${resource.name}"` };
-    }
+    // mainly to help E2E tests distinguish direct connections from other tree items
+    this.accessibilityInformation = {
+      label: `${this.resource.connectionType}: connection "${resource.name}"`,
```

Comment:
I don't know if we necessarily need the connection name in here as opposed to just `[CCLOUD|DIRECT|LOCAL] connection`